apps/condo/domains/contact/access/ContactRole.js

```diff
@@ -0,0 +1,61 @@
+// @ts-nocheck
+/**
+ * Generated by `createschema contact.ContactRole 'name:Text'`
+ */
+
+const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+const {
+    queryOrganizationEmployeeFor,
+    checkOrganizationPermission,
+} = require('@condo/domains/organization/utils/accessSchema')
+const get = require('lodash/get')
+const { getById } = require('@core/keystone/schema')
+const { STAFF } = require('@condo/domains/user/constants/common')
+
+async function canReadContactRoles ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    if (user.isAdmin || user.isSupport) return {}
+
+    if (user.type !== STAFF) return false
+
+    return {
+        OR: [
+            { organization_is_null: true },
+            { organization: queryOrganizationEmployeeFor(user.id) },
```

Comment:
what about employees from related `from` organization?
they have access to read and manage the contacts of the organization